Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.mm

```diff
@@ -42,34 +42,78 @@ @implementation _WKWebExtension
 
 #if ENABLE(WK_WEB_EXTENSIONS)
 
-- (instancetype)initWithAppExtensionBundle:(NSBundle *)appExtensionBundle
++ (instancetype)extensionWithAppExtensionBundle:(NSBundle *)appExtensionBundle
+{
+    NSError * __autoreleasing internalError;
+    auto result = WebKit::WebExtension::create(appExtensionBundle, &internalError);
+
+    if (internalError)
+        return nil;
```

Comment:
Do we have any way to log these? Do we want to?

Reply:
Nit: This could be `return internalError ? nil : result->wrapper();`

Reply:
We have `RELEASE_LOG(Extensions, "...")`.